src/lib/tenant/configs/ui/uniswap.ts

```diff
@@ -218,6 +218,22 @@ export const uniswapTenantUIConfig = new TenantUI({
       name: "info/governance-charts",
       enabled: true,
     },
+    {
+      name: "sponsoredVote",
+      enabled: true,
+      config: {
+        domain: "Uniswap Governor Bravo",
```

Comment:
We will need to update this to handle different contract names between Sepolia and Mainnet.